src/app/inventory/InventoryItem.tsx

```diff
@@ -43,15 +42,14 @@ export default function InventoryItem({
   tag,
   notes,
   searchHidden,
-  wishListsEnabled,
-  inventoryWishListRoll,
+  wishlistRoll,
   ignoreSelectedPerks,
   onClick,
   onShiftClick,
   onDoubleClick,
   innerRef,
 }: Props) {
-  const uiWishListRoll = wishListsEnabled ? toUiWishListRoll(inventoryWishListRoll) : undefined;
+  const uiWishListRoll = useMemo(() => toUiWishListRoll(wishlistRoll), [wishlistRoll]);
```

Comment:
Forgot this was here - can we just use one type instead of converting them back and forth?

Reply:
Oh I see, this isn't even another type, it just returns `inventoryWishListRoll.isUndesirable ? UiWishListRoll.Bad : UiWishListRoll.Good`. Definitely not worth `useMemo`, potentially worth baking into the roll generation process itself or flattening into this component.